materialspinner/src/main/java/xyz/arifz/materialspinner/MaterialSpinner.kt

```diff
@@ -204,4 +219,35 @@ class MaterialSpinner : TextInputLayout {
         autoCompleteTextView.onItemClickListener = listener
     }
 
+    fun setHintAsteriskColor(color: Int) {
+        val len = hintForRed.length
+        val sb = SpannableStringBuilder(hintForRed)
+        val asteriskColor = ForegroundColorSpan(color)
+        if (len != 0) {
+            sb.setSpan(asteriskColor, len - 1, len, Spannable.SPAN_INCLUSIVE_INCLUSIVE)
+            super.setHint(sb)
+        }
+    }
+
+    fun setHintFontFamily(fontFamily: Int) {
+        fontFamily.let { ResourcesCompat.getFont(context, it) }.also { typeface = it }
+    }
+
+    fun setBoxWidth(size: Int) {
+        boxStrokeWidth = size
+        boxStrokeWidthFocused = size
+    }
+
+    fun setTextFontFamily(fontFamily: Int) {
+        autoCompleteTextView.typeface = fontFamily?.let { ResourcesCompat.getFont(context, it) }
+    }
+
+    fun setTextColor(textColorCode: String) {
+        autoCompleteTextView.setTextColor(Color.parseColor(textColorCode))
+    }
+
+    fun setTextSize(fontSizeDp: Int) {
```

Comment:
are we asking font size in `dp` format or `sp` format? if we are asking `sp` format than I thing this variable should be rename to `fontSizeSp`